editor/project_manager/project_dialog.cpp

```diff
@@ -245,6 +245,35 @@ void ProjectDialog::_validate_path() {
 			}
 		}
 	}
+
+	// Check if the target path is a subdirectory of original when duplicating
+	if (mode == MODE_DUPLICATE) {
+		String base_path = original_project_path;
+		String duplicate_target = target_path;
+
+		// Ensure the paths end with a slash
+		if (!base_path.ends_with("/")) {
+			base_path += "/";
+		}
```

Comment:
You can instead remove slashes.
```suggestion
	base_path = base_path.trim_suffix("/");
```